keras_core/ops/image.py

```diff
@@ -55,7 +55,35 @@ def compute_output_spec(self, image):
 def resize(
     image, size, method="bilinear", antialias=False, data_format="channels_last"
 ):
-    # TODO: add docstring
+    """Resize images to size using the specified method.
+
+    Args:
+        image: Input image or batch of images. Must be 3D or 4D.
+        size: Size of output image in (height, width) format.
+        method: Interpolation method. Defaults to 'bilinear'.
+        antialias: Whether to use an antialiasing filter when downsampling an
+            image.
+        data_format: Format of the image tensor. Defaults to 'channels_last'.
+
+    Returns:
+        Resized image or batch of images.
+
+    Examples:
+
+    >>> x = np.random.random((2, 4, 4, 3)) # batch of 2 RGB images
+    >>> y = keras_core.ops.image.resize(x, (2, 2))
+    >>> y.shape
```

Comment:
Print shape (same below)

Reply:
That depends on the backend, if it is TensorFlow it'll output `TensorShape` etc. Guess it is best to remove `y.shape` then?

Reply:
Is it OK to put symbolic outputs here, like: `(2, 2, 2, 3)` etc?

Reply:
Yes, it should just be indicative.